src/components/ContentWidgets/ToDoList/ToDoList.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { ContentSingularData } from "../../_data/ContentSingularData";
+
+/** 
+ * A to-do list widget
+ * 
+ * Last Modified
+ * Allison Lee
+ * June 28, 2019
+**/
+export const ToDoList: React.FC<ContentSingularData> = ({
+    toDoList
```

Comment:
You would want to make the variable `toDoList_something`, probably not just `toDoList`. If anything it would be `toDoList_list`. Repetitive yes, but it follows the semantics of the existing props.